src/skin/js/firstRun.js

```diff
@@ -15,4 +15,18 @@ $(function () {
       });
     }
   });
+
+  // Nudge Chrome users to pin the extension to their toolbar
+  if (navigator && navigator.userAgent && navigator.userAgent.match(/chrome/i)) {
```

Comment:
Our goal here is to not show the nudge in browsers where we can [auto-pin using the `default_area` property](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action#syntax), right? That's only Firefox. All Chromium-based browsers (Edge, Opera, Brave, ...) should also get this prompt.

We should avoid checking the UA whenever possible because users change their UA or blank it out among other reasons.

`browser.browserAction.getUserSettings()` exists in Firefox and will async return an object with `isOnToolbar` set to `true`.
`chrome.action.getUserSettings()` exists in MV3 Chrome and will async return an object with `isOnToolbar` set to `false`.

We should probably use this to decide whether to show the nudge or not.

Reply:
I thought this change was only for Chrome users but this is a good idea for how to target all browsers except for Firefox! Thank you! 

One complication with including Edge, Opera, and Brave is that the icons that users need to click are a little different in each browser. For example, Opera has a box instead of a puzzle piece as the extension icon in the toolbar. To pin on Edge, users select an eye icon instead of a pin icon. So we might need to detect which browser the page is on and load a different image for each browser. That's what Daly did in [this PR](https://github.com/EFForg/privacybadger/pull/2812/files#diff-4488bb389c2092a3e38367777d8fc45e9a422f51943ff728a8901cb490716a3a). 

Let's discuss what approach to take for different browsers when we talk about a new design proposal?